_component_design/status-labels.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Status Labels
+parent: Notifications
+layout: component
+category: Components
+usage: >
+    Status Labels are wide tags that include short, brief, and helpful text communication regarding the progress of our users accounts.
+status: stable
+resource: true
+preview-image: preview-images/status-labels.svg
+last-modified: 2017-12-01
+helix-ui-css: true
+helix-ui-javascript: false
+---
+
+{% include toc.html %}
+
+## Introduction
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### When to use
+Use status labels to communicate the current state of an object or process. For example, to communicate the status of a support ticket, or current health of a device.
```

Comment:
Second sentence isn't a sentence. Change to:
```markdown
Use status labels to communicate the current state of an object or process, such as the status of a support ticket or the current health of a device.
```

Reply:
updated.